tasks/skylab/LoomUtils.wdl

```diff
@@ -89,7 +89,7 @@ task OptimusLoomGeneration {
 
     Int preemptible = 3
     Int disk = 200
-    Int machine_mem_mb = 18
```

Comment:
would it make sense to make it 16000?

Reply:
Sure. Do you want me to change all instance of 18GB in this WDL to 16000MB?

Reply:
I think it makes more sense to be 16000. Thank you!